server/migrations/20230627154012-add-tag-scene.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('t_scene', 'tags', {
```

Comment:
I thought we said we were doing it the relational way?

With this option, it'll hard (if not impossible) to fetch existing tags in SQL

Reply:
Fixed